### ISSUE TITLE
Bucket missing in a11y view

### ISSUE THREAD

**Issue by arouinfar** [phetsims/forces-and-motion-basics#424]
The Acceleration Screen swaps out the trash can for a bucket of water:
<img width="485" height="104" alt="Image" src="https://github.com/user-attachments/assets/1c136e7c-f2c3-4753-bb48-d46274058fb4" />

The bucket is missing its accessible name, but is interactive with alt input. Not sure if there is anything else to hook up or not.
<img width="459" height="106" alt="Image" src="https://github.com/user-attachments/assets/e9eb4d49-575a-4d40-b160-7778ed3b7522" />

@samreid please take a look and see if anything beyond an accessible name is needed.
- [ ] Accessible Name: Bucket of Water
